providers/curation/github.js

```diff
@@ -109,13 +109,38 @@ class GitHubCurationService {
     // Github requires name/email to set committer
     if ((info.name || info.login) && info.email)
       fileBody.committer = { name: info.name || info.login, email: info.email }
+    else {
+      logger.info('Github requires name/email to set committer, info.email = ', info.email)
+    }
```

Comment:
You can compact the code without curly braces like `else logger.info('Github requires name/email to set committer, info.email = ', info.email)`